controllers/DebugController.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
All files named 'debug' is temporary and will be deleted by finish the development.